go/engine/saltpack_encrypt_test.go

```diff
@@ -57,7 +60,7 @@ func TestSaltPackEncryptSelfNoKey(t *testing.T) {
 
 	sink := libkb.NewBufferCloser()
 	arg := &SaltPackEncryptArg{
-		Recips: []string{"t_alice", "t_bob+kbtester1@twitter", "t_charlie+tacovontaco@twitter"},
+		Recips: []string{"t_tracy+t_tracy@rooter", "t_george", "t_kb+gbrltest@twitter"},
```

Comment:
It turns out this test was passing, but for the wrong reason. It fails because the t_(alice|bob|charlie) don't have device keys, only PGP keys.  Fixed, these test users do.